### ISSUE TITLE
[Adaptive Card iOS - Input.ChoiceSet.ErrorMessage.json]: Voice over is announcing label with all radio buttons present under label 'Required Input.ChoiceSet label (expanded)'. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8403]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice Over
4. Navigate to "V1.3" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Input.ChoiceSet.ErrorMessage.json' under 'Elements' list item and activate it.
7. Card will appear.
8. Navigate to 'Option 1' control.
9. Observe 'voice over' issue 

**Actual Result:**
Voice over is announcing label with all radio buttons present under label 'Required Input.ChoiceSet label (expanded)'.
Voice over is also not announcing position with radio button.

Voice over is announcing first radio button as 'Required Input.ChoiceSet label (expanded) Option 1 button' and second radio button as 'Required Input.ChoiceSet label (expanded) Option 2 button' present under label 'Required Input.ChoiceSet label (expanded)'.

**Expected Result:**
Voice over should announce label with first radio button when user enter list of radio button in forward navigation or backward navigation
Voice over should announce position with all radio button.

Voice over should announce first radio button as 'Required Input.ChoiceSet label (expanded) Option 1 button radio button 1 of 2 non selected' and announce second radio button as 'Option 2 radio button 2 of 2 not selected' present under label 'Required Input.ChoiceSet label (expanded)'.

**Notes:**
1.Issue is repro'ing with all radio buttons present in this card.
2. v1.5-Tests-Input.ChoiceSet.FilteredStyle.TestCard.json- Voice over is announcing label with all checkboxes.(Please find attachment)
3. This issue is repro throughout the application for the same type of controls.

**User Experience:**
Voice over user will be impacted if Voice over is announcing label with all radio buttons present under label 'Required Input.ChoiceSet label (expanded)'.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

<img width="563" alt="Note-Voice over is announcing label with all checkboxes" src="https://user-images.githubusercontent.com/95913748/227834844-c6306869-1826-4195-be56-c1b3db07c93d.png">

https://user-images.githubusercontent.com/95913748/227834918-b2da6e12-386a-4882-b5d3-304176a8f9e0.mov

### Card JSON

```JSON
NA
```
